### PR TITLE
docs(tab): add additional examples

### PR DESCRIPTION
**Pull request by whiplashwebb**
Our tab examples feature very reasonable content. Unfortunately, we do not live in a reasonable world and our components should work as well as can be expected even when our users do unreasonable things. This PR adds some examples to the docs which expose a few edge case bugs in our themes.

## Proposed Changes

- Adds tabs to makes most examples show a mixture of tabs with and without icons to test for height bugs.
- Adds one tab with content which is much longer with no BRs to test wrapping etc when height changes.

### Bootstrap

Tab height issues at medium/large size

Screenshot 2025-04-14 at 11 22 25 AM

Vertical tab collapses with large content

https://github.com/user-attachments/assets/10644340-05ea-4b04-be7b-648f804d7413

### Bulma

Tab height issues (not all shown)

Screenshot 2025-04-14 at 11 27 14 AM

Vertical tab wraps with large content

https://github.com/user-attachments/assets/4fbf184f-6666-4d7f-b092-1a24517764bc

I plan to do a theme Bulma PR to address those issues. I may be able to do bootstrap as well if time permits.

### CODE REVIEW

File: packages/oruga/src/components/tabs/examples/expanded.vue [hunk 1]
```diff
@@ -3,7 +3,10 @@
         <o-tabs expanded>
             <o-tab-item label="Pictures" icon="images" />
             <o-tab-item label="Music" icon="music" />
+            <o-tab-item label="Words" />
             <o-tab-item label="Videos" icon="video" />
         </o-tabs>
     </section>
 </template>
+<script setup lang="ts">
```
[the remark this answers is not in the document]
Comment (whiplashwebb) [reply packages/oruga/src/components/tabs/examples/expanded.vue:11]: weird, did my IDE add that? i have no idea why that's there lol. removed.

File: packages/oruga/src/components/tabs/examples/position.vue [hunk 2]
```diff
@@ -3,19 +3,24 @@
...
         <o-tabs type="boxed" position="right">
             <o-tab-item label="Pictures" icon="images" />
             <o-tab-item label="Music" icon="music" />
+            <o-tab-item label="Words" />
             <o-tab-item label="Videos" icon="video" />
         </o-tabs>
     </section>
 </template>
+<script setup lang="ts">
+</script>
```
[the remark this answers is not in the document]
Comment (whiplashwebb) [reply packages/oruga/src/components/tabs/examples/position.vue:26]: don't know where this came from either, removed.
